Formula/pplacer.rb

```diff
@@ -0,0 +1,34 @@
+class Pplacer < Formula
+  # Matsen_2010: "https://dx.doi.org/10.1186/1471-2105-11-538"
+  desc "Place query sequences on a fixed reference phylogenetic tree"
+  homepage "https://matsen.fhcrc.org/pplacer/"
+
+  # We use binaries to avoid compiling OCaml code
+  if OS.mac?
+    url "https://github.com/matsen/pplacer/releases/download/v1.1.alpha17/pplacer-Darwin-v1.1.alpha17.zip"
+    sha256 "db1ac64e1bc9b4d24d17ee2e388c061c283ca9fbec075e022bdeaad1adc6d41c"
+  else
+    url "https://github.com/matsen/pplacer/releases/download/v1.1.alpha17/pplacer-Linux-v1.1.alpha17.zip"
+    sha256 "3dc8e20fa8642d01daadde5bf9e36df2c180abec8f85c0b2f296f7852b63537c"
+  end
+
+  depends_on "gsl"
+  depends_on "zlib" unless OS.mac?
+
+  def install
+    binaries = ["pplacer", "guppy", "rppr"]
+    if OS.mac?
+      binaries.each do |bin|
+        MachO::Tools.change_install_name bin, "/usr/local/lib/libgsl.0.dylib", "#{Formula["gsl"].lib}/libgsl.dylib"
+        MachO::Tools.change_install_name bin, "/usr/local/lib/libgslcblas.0.dylib", "#{Formula["gsl"].lib}/libgslcblas.0.dylib"
+        MachO::Tools.change_install_name bin, "/usr/local/lib/gcc/5/libgcc_s.1.dylib", "/usr/lib/libgcc_s.1.dylib"
```

Comment:
Does using `/usr/lib/libgcc_s.1.dylib` work? Should it `depends_on "gcc@5"` instead?

Reply:
Are the `gsl` patches needed on Linux?

Reply:
I ran it on some of my own data, and it seemed to run fine. The binaries used to work out of the box on `macOS`, but something must have changed is Mojave.

Reply:
Not sure about Linux, I'll take a closer look.

Reply:
What does `brew linkage pplacer` report on macOS and Linux?

Reply:
macOS
```
 ❯ brew linkage pplacer
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libgcc_s.1.dylib
  /usr/lib/libsqlite3.dylib
  /usr/lib/libz.1.dylib
Homebrew libraries:
  /usr/local/opt/gsl/lib/libgsl.dylib (gsl)
  /usr/local/opt/gsl/lib/libgslcblas.0.dylib (gsl)
```

Reply:
just installing on linux now...

Reply:
Linux
```
$ brew linkage pplacer
Dependencies with no linkage:
  zlib
```

That's odd.

Reply:
I wonder if they statically compiled everything into the binary.

Reply:
I've removed the dependency on `gsl` and `zlib` for Linux.

Reply:
The upstream authors probably did that because on macOS you can ask users to install Homebrew, but on Linux that's less typical, so they just linked statically.